### ISSUE TITLE
请问我单独监听mysql，怎么没反应呢

### ISSUE THREAD

**Issue by fliyu** [kevwan/tproxy#12]
<img width="566" alt="image" src="https://user-images.githubusercontent.com/110647061/201029984-6fa18d00-a554-4be9-9899-34655f8149cd.png">
arm64框架，监听mysql，开了个服务调用
<img width="604" alt="image" src="https://user-images.githubusercontent.com/110647061/201030276-97f2c888-4f7a-4827-b3a0-40511786b045.png">

**Comment by kevwan** [comment]
Connect to port 3307 in your program?

**Comment by fliyu** [comment]
no,my program port is 8888,I want to check the connection status of MySQL
if I remove -p param, It didn't respond
```
$ tproxy -r 127.0.0.1:3306 -s -q 
09:30:26.794 Listening on 127.0.0.1:60785...
```

**Comment by kevwan** [comment]
Listening on port 60785
